### ISSUE TITLE
Reader not Loading

### ISSUE THREAD

**Issue by SWT92** [LNReader/lnreader#266]
I downloaded 2 novel, was reading first novel and currently unable to open the offline reader.

First novel : Unable to open and read normally
Second novel : Able to open and read normally

Work around : using history to continue reading the novel. 

What it shows : 
![Screenshot_2022-05-23-03-19-35-25_3c1c179116822eec69c21805387f0489](https://user-images.githubusercontent.com/20933151/169712251-721092e6-4db6-4257-b772-bee479af4b22.jpg)

**Comment by rajarsheechatterjee** [comment]
Can you provide the source, novel and chapter name and steps to reproduce it.

**Comment by SWT92** [comment]
> Can you provide the source, novel and chapter name and steps to reproduce it.

Source : ReaLightNovel(Not very sure)
Novel : My Vampire System
Chapter : 500++ 

Steps to reproduce : 
Open "My Vampire System" Novel via the main page(Library)
Stuck at splash screen

Quit the app

Open another novel via the Library page and went to the chapter selection page. 

Do you need a video of it?

**Comment by rajarsheechatterjee** [comment]
It's probably happening because [ReadLightNovel is currently  broken](https://github.com/LNReader/lnreader-sources/issues/226). 

Does it also happen for novels from other sources?

**Comment by SWT92** [comment]
> It's probably happening because [ReadLightNovel is currently broken](https://github.com/LNReader/lnreader-sources/issues/226).
> 
> Does it also happen for novels from other sources?

Nope. It does not happen for my other novel that i downloaded via other source. 

It doesnt make sense since the novel I read is downloaded offline, even if the source server is down or broken, it shouldnt show the current bug? 

Anyway please at a look at the video. 
https://youtube.com/shorts/jU7rBBJRDeg?feature=share

**Comment by rajarsheechatterjee** [comment]
> It doesnt make sense since the novel I read is downloaded 8s currently in offline mode, even if the source server is down or broken, it shouldnt show the current bug?

Yep, I should add better error handling on that screen so that even if a source is broken it should not make the app crash.

**Comment by SWT92** [comment]
> > It doesnt make sense since the novel I read is downloaded 8s currently in offline mode, even if the source server is down or broken, it shouldnt show the current bug?
> 
> Yep, I should add better error handling on that screen so that even if a source is broken it should not make the app crash.

Oh wow... You really have got great understanding... 

Thank you so much!!!

**Comment by nyagami** [comment]
This issue might be resolved in v2 app
